src/app/components/mnoe-users-local-list/mnoe-users-local-list.coffee

```diff
@@ -78,8 +78,88 @@
           MnoErrorsHandler.processServerError(error)
       ).finally(-> user.isSendingInvite = false)
 
+    scope.tlKeyFromUserRole = (role) ->
+      enTranslationTable = $translate.getTranslationTable('en-AU')
+      _.findKey(enTranslationTable, _.partial(_.isEqual, role))
```

Comment:
This will always return the translation for the `en-AU` locale.

Why don't you simply use $translate API to retrieve the corresponding translations?

Reply:
For the role, we display the value from MnoHub. The value stored there is in English.
Therefore, this method returns the key associated to that value, so that I can display the translated value in the frontend. 
=> "Member" returned by MnoHub, will be displayed "Membre" in French